data/json/items/armor/jewelry.json

```
@@ -1293,6 +1300,14 @@
         "color": "yellow",
         "weight": 5
       },
+      {
+        "id": "saint_necklace",
+        "expand_snippets": true,
+        "name": { "str": "Saint Medallion" },
+        "description": "A necklace made of a fine gold chain with a medallion bearing the image of a Catholic Saint.  The back reads \"<catholic_saints>\".",
```

I _think_ "saint" should be lowercased when not used as a title
```suggestion
        "description": "A necklace made of a fine gold chain with a medallion bearing the image of a Catholic saint.  The back reads \"<catholic_saints>\".",
```